src/it/java/dev/jbang/it/BaseIT.java

```diff
@@ -45,7 +45,12 @@ protected Path baseDir() {
 	public static List<String> prefixShellArgs(List<String> cmd) {
 		List<String> list = new ArrayList<>(cmd);
 		if (Util.isWindows()) {
-			list.addAll(0, Arrays.asList("cmd", "/c"));
+			if ("true".equals(System.getProperty("jbang.it.usePowershell"))) {
```

Comment:
could just be `Boolean.getBoolean("jbang.it.usePowershell")`

Reply:
Always forget about those, it's such a weird place to put those methods.